packages/ui/src/Popover/index.tsx

```diff
@@ -122,19 +94,10 @@ export const Popover = ({ children, onClose, isOpen }: PopoverProps) => {
 
 export interface PopoverTriggerProps {
   children: ReactNode;
-  /**
-   * Change the default rendered element for the one passed as a child, merging
-   * their props and behavior.
-   *
-   * Uses Radix UI's `asChild` prop under the hood.
-   *
-   * @see https://www.radix-ui.com/primitives/docs/guides/composition
-   */
-  asChild?: boolean;
```

Comment:
Removed `asChild` prop from the `Popover.Trigger` and `DropdownMenu.Trigger`. Since we do not users to change the component styles, it makes to sense to allow `asChild={false}` – it will always be an unstyled HTML-element

Reply:
OK. I could see there being a use case where users want to just have plain text as the trigger. But maybe that's not even a good idea. So, no strong opinion here.